src/languages/en.ts

```diff
@@ -1921,6 +1921,8 @@ export default {
             customersDescription: 'Choose whether to import customers/projects and see where customers/projects are displayed.',
             locationsDescription: 'Choose whether to import locations, and see where locations are displayed.',
             taxesDescription: 'Choose whether to import tax rates and tax defaults from your accounting integration.',
+            taxesJournalEntrySwitchNote:
```

Comment:
@teneeto, could you remind me where you got this UI copy? 😄

Reply:
@hayata-suenaga in a moment

Reply:
@hayata-suenaga here was the [last discussion](https://github.com/Expensify/App/issues/41042#issuecomment-2083614043) between @zanyrenney and @trjExpensify that lead to that UI 😉

Reply:
thank you for confirming! 🙇